### DISCUSSION TITLE
Making a Scene in spoke I can't make it appear in the featured scenes

### DISCUSSION THREAD

**Discussion by elmau** [mozilla/hubs#3284] [Hubs Cloud Questions] [answered]
How do I do it?

**Comment by BonzoMartini** [comment, marked as answer]
See "How to move a Scene to the Featured Scenes in Hubs Cloud?? #3282"

**Reply by elmau** [reply]
Doesn't work for me

**Reply by robinkwilson** [reply]
Have you published the scene in spoke first?
Does it appear in the Pending Scenes or Approved Scenes menus?
Can you describe the steps you've taken?

**Reply by robinkwilson** [reply]
Please `unmark as answer` when more info is provided! Thanks!
